src/main/resources/userEmailFragment-sessionClosedAdditionalInstructionsFragment.html

```diff
@@ -1,4 +1 @@
-<p>
-  <strong>No action is required if you have already submitted.</strong>
-  <br><span style="background-color: #FFFF00">Any request for deadline extensions should be sent to the instructors of the course,</span> not to the TEAMMATES support team.
-</p>
+<li><strong>If already submitted:</strong> No action is required.</li>
```

Comment:
I don't think it's right to have one email fragment with only this line, and especially when this line is also duplicated in some other templates.
Either that, or the file name should be changed to reflect on the current usage.

Reply:
@wkurniawan07 
I kept just this line in this file because otherwise there was a unnecessary line break between "If already submitted " bullet point and the rest of the bullet points. (And I did not include this line in the `feedbackSession` file either as that template is being used to generate other emails as well. Like `feedbackSessionOpening` email. ) 

Yes, this line is also in `userEmailTemplate-feedbackSessionClosing` file.
I had one question when I was rephasing the email Templates and that was, `userEmailTemplate-feedbackSessionClosing` and `userEmailTemplate-feedbackSession` had almost the same content except two lines : 
```
 To submit feedback for the above session, please go to this Web address: ${submitUrl} //This is the feedbackSessionClosing template
 To ${feedbackAction} your feedback for the above session, please go to this Web address: ${submitUrl} //This is the feedbackSubmission template

If already submitted: No action is required  //This is the feedbackSessionClosing template
${additionalInstructions} //This is the feedbackSubmission template
```
**My question is that :** Is there a reason for two separate files?<br>I mean can't we use the `feedbackSession` file to generate the `feedbackSessionClosing` file using this `userEmailFragment-sessionClosedAdditionalInstructionsFragment.html ` file to provide the line `If already submitted: No action is required.` We can also change the name of this file to match the usage as you suggested. (But right now, I don't know how to do this, It may take a little longer than usual to resolve this like a few days, I hope that is not a problem :sweat_smile: )

Reply:
I modified the `EmailGenerator.java` to use `userEmailTemplate-feedbackSession.html` file as the template for `sessionClosingEmailForInstructor` and `sessionClosingEmailForStudent`, so that we don't need a different template for the closing emails. But I haven't commited anything yet and haven't changed the `userEmailFragment-sessionClosedAdditionalInstructionsFragment.html` , as I want your suggestions on this first 

>Is there a reason for two separate files?
I mean can't we use the feedbackSession file to generate the feedbackSessionClosing file using this userEmailFragment-sessionClosedAdditionalInstructionsFragment.html file to provide the line If already submitted: No action is required. We can also change the name of this file to match the usage as you suggested.